gather-scheduler.js

```diff
@@ -28,6 +28,8 @@ class GatherScheduler {
   static run(gatherers, options) {
     const driver = options.driver;
     const url = options.url;
+    const loadPage = options.flags.loadPage;
```

Comment:
we need to make sure these have defaults, i it is a required value we should throw